src/craftdef.cpp

```diff
@@ -157,32 +155,27 @@ static bool craftGetBounds(const std::vector<std::string> &items, unsigned int w
 	bool success = false;
 	unsigned int x = 0;
 	unsigned int y = 0;
-	for(std::vector<std::string>::const_iterator
+	for (std::vector<std::string>::const_iterator
 			i = items.begin();
-			i != items.end(); i++)
```

Comment:
It doesn't prevent errors per se, it just makes things clearer that it's not an counting integer.  I use it as well.